cmd/gossamer/README.md

```diff
@@ -0,0 +1,190 @@
+# Gossamer `cmd` Package
+
+This package encapsulates the entry point to Gossamer - it uses the popular
+[`cli` package from `urfave`](https://github.com/urfave/cli/blob/master/docs/v1/manual.md) to expose a command-line
+interface (CLI). The Gossamer CLI accepts several subcommands, each of which is associated with an "action"; these
+subcommands and their corresponding actions are defined in [`main.go`](main.go). When the Gossamer CLI is executed
+without a subcommand, the `gossamerAction` is invoked.
+
+## Actions & Subcommands
+
+What follows is a list of the Gossamer subcommands, as well as an overview of some of the flags/parameters they accept.
+The flags/parameters that the Gossamer CLI supports are defined in [`flags.go`](flags.go). For an exhaustive reference
+of the Gossamer CLI capabilities, follow [the installation instructions](../../README.md#installation) and execute
+`./bin/gossamer --help`.
+
+### Default Command
+
+This the default Gossamer execution method, which invokes the `gossamerAction` function defined in
+[`main.go`](main.go) - it will launch a Gossamer blockchain client. The details of how Gossamer orchestrates a
+blockchain client are [described below in the Client Components section](#client-components).
+
+- `basepath` - the path to the directory where Gossamer will store its data
```

Comment:
It's not totally clear these are flags to me, reading from the top. Maybe add a sentence before the bullet points or prefix each with `--` (I prefer the second for copy pasta purposes but that's subjective)

Reply:
You had me at "copy pasta" 😋